Dockerfile.repo

```diff
@@ -54,14 +53,14 @@ RUN \
     --with-apreq2-config=/usr/bin/apreq2-config && \
   make && \
   make install
-  
-RUN \
-  chkconfig httpd off
 
 RUN wget https://github.com/stevengj/nlopt/archive/v2.7.1.tar.gz && \
     tar -xzvf v2.7.1.tar.gz && \
     cd nlopt-2.7.1 && \
     cmake . && make && sudo make install
+ENV LD_LIBRARY_PATH /nlopt-2.7.1:${LD_LIBRARY_PATH}
+# Run ldconfig to update the shared library cache
+RUN ldconfig
```

Comment:
Needed this to have nlopt be found when installing the package https://cran.r-project.org/web/packages/nloptr/index.html

Not sure why it wasn't needed before.